sortinghat/core/api.py

```diff
@@ -771,59 +813,60 @@ def _merge_enrollments(trxl, from_uid, to_uid):
 
         return to_uid
 
-    def _merge_profiles(from_uid, to_uid):
-        """Merge the profiles from two `UniqueIdentity` objects"""
-        if from_uid.profile.is_bot or to_uid.profile.is_bot:
-            to_uid.profile.is_bot = True
-        if not to_uid.profile.name:
-            to_uid.profile.name = from_uid.profile.name
-        if not to_uid.profile.email:
-            to_uid.profile.email = from_uid.profile.email
-        if not to_uid.profile.gender:
-            to_uid.profile.gender = from_uid.profile.gender
-        if not to_uid.profile.gender_acc:
-            to_uid.profile.gender_acc = from_uid.profile.gender_acc
-        if not to_uid.profile.country:
-            to_uid.profile.country = from_uid.profile.country
+    def _merge_profiles(from_uids, to_uid):
+        """Merge the profiles from `UniqueIdentity` objects"""
+
+        for from_uid in from_uids:
+            if from_uid.profile.is_bot or to_uid.profile.is_bot:
+                to_uid.profile.is_bot = True
```

Comment:
Wouldn't be better to limit this to `if not to_uid.profile.is_bot and from_uid.profile.is_bot`?

Reply:
You are totally right, now the condition is redundant. I'm changing this.